ui/packages/app/web/src/components/ui/Card.tsx

```diff
@@ -21,10 +21,12 @@ const Card = ({children, className = ''}: CardProps) => {
   );
 };
 
-const Header = ({children}: {children: React.ReactNode}) => {
+const Header = ({children, className}: {children: React.ReactNode; className?: string}) => {
```

Comment:
Should we also assign a default parameter value for `className` to be `''` as it is an optional, and we have a `className.length` below?